spec/10-expressions.md

```diff
@@ -2053,7 +2051,7 @@ otherwise, the type of the result is `float`.
 
 The binary `%` operator produces the remainder from dividing the left-hand
 operand by the right-hand one. If the type of both operands is not `int`,
-their values are converted to that type. The result has type `int`.
+their values are converted to that type. The result has type `int`. If the right-hand operand has value zero, an exception of type `DivisionByZeroError` is thrown.
```

Comment:
Independently of this patch: It would be nice to clarify whether PHP uses "mod" or "rem" behavior for negative numbers.